app/javascript/components/supplemental_application/sections/LeaseInformationInputs.js

```diff
@@ -33,10 +33,16 @@ const getTotalMonthlyRent = (values) => {
   }
 }
 
+const toggleNoPreferenceUsed = (formApi) => {
+  if (formApi.touched.preference_used) { formApi.values.no_preference_used = !formApi.values.preference_used }
```

Comment:
values should not be modified directly. Instead `formApi.setValue(field, value)` should be used.
react-form refs for our version: https://dazzling-keller-a8107a.netlify.com/#/

Reading from `formApi.value` is ok 👍

Reply:
@fddayan Good catch, I will change to use `setValue` instead.